### DISCUSSION TITLE
Openweather API dosen't work

### DISCUSSION THREAD

**Discussion by RasputinBerry** [flobz/psa_car_controller#833] [Q&A]
PSACC runs in a QNAP NAS Container Station and works (fine), this ist the config.json ...

{
    "abrp": {
        "abrp_enable_vin": [],
        "token": ""
    },
    "client_id": "07364655-xxxxxxxxxxxxxxxxxxxx5iP5cN8cJ8",
    "co2_signal_api": "gxkjxxxxxxxxpS",
    "country_code": "DE",
    "customer_id": "OP-ACNxxxxxxxxx051",
    "proxies": {
        "http": "",
        "https": ""
    },
    "realm": "clientsB2COpel",
    "refresh_token": "88b2b0f1-3976-xxxxxxxxxx33e3a5e9c7a7",
    "remote_refresh_token": "634c8449-xxxxxxxxxxxx-a69456e75ab0",
    "weather_api": "9dfc64axxxxxxxxx3a1c6a44a1"
}

... but I get this error message ...

`2024-03-27 14:06:13,566 :: ERROR :: Unable to get temperature from openweathermap :
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/psa_car_controller/psacc/utils/utils.py", line 20, in get_temp
    temp = weather_res_json.get("current", weather_res_json.get("main"))["temp"]
TypeError: 'NoneType' object is not subscriptable`

I have no idea where the error could be ...

EDIT: The API in the browser works .... https://api.openweathermap.org/data/2.5/weather?lat=xxx&lon=yyyy&appid=xxxxxxxxxxxxxx

**Comment by 5kat3R** [comment]
Could this be the problem?
https://github.com/flobz/psa_car_controller/pull/612#issuecomment-1902250380

**Comment by RasputinBerry** [comment]
... at least "my" error message looks similar to "yours" ... Maybe @flobz is reading here and thinks about the wish ... I hope so ...
